### PR TITLE
Fixes to arxiv integration doc

### PR DESCRIPTION
**Pull request by abidlabs**
Makes a few fixes to the ArXiv integration docs:

* Removes references to linking Spaces to datasets since that doesn't work with the ArXiv integration right now
* Added link to announcement blog post
* Added screenshot of how the arxiv integration looks

### CODE REVIEW

File: docs/hub/spaces-add-to-arxiv.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+learning research without writing any code. [Hugging Face and ArXiv have collaborated](https://huggingface.co/blog/arxiv) 
 to embed these demos directly along side papers on ArXiv!
 
-Thanks to this integration, users can now find the most popular demos for a paper on its arXiv abstract page. For example, if you want to try out demos of the LayoutLM document classification model, you can go to [the LayoutLM paper's arXiv page](https://arxiv.org/abs/1912.13318), and navigate to the demo tab. You will see open-source demos built by the machine learning community for this model.
+Thanks to this integration, users can now find the most popular demos for a paper on its arXiv abstract page. For example, if you want to try out demos of the LayoutLM document classification model, you can go to [the LayoutLM paper's arXiv page](https://arxiv.org/abs/1912.13318), and navigate to the demo tab. You will see open-source demos built by the machine learning community for this model, which you can try out immediately in your browser:
+
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/layout-lm-space-arxiv.gif)
```
Comment (osanseviero) [inline docs/hub/spaces-add-to-arxiv.md:9]: Was this gif deleted? https://huggingface.co/datasets/huggingface/documentation-images/commit/27b5026dd00e2c6571651b201d59611608772f71

**Review by osanseviero — APPROVED** [review]
Thanks a lot for the iteration! :fire: 

The gif is not in the dataset still, so that needs to be added first

File: docs/hub/spaces-add-to-arxiv.md (hunk 1 above)
Comment (abidlabs) [reply docs/hub/spaces-add-to-arxiv.md:9]: Haha you're too quick! Just fixed, the GIF was uploaded in the wrong location

Comment (osanseviero) [reply docs/hub/spaces-add-to-arxiv.md:9]: Nice gif!

**Pull request merged**